features/lwipstack/lwip/src/api/lwip_tcpip.c

```diff
@@ -253,6 +253,7 @@ tcpip_inpkt(struct pbuf *p, struct netif *inp, netif_input_fn input_fn)
 
   msg = (struct tcpip_msg *)memp_malloc(MEMP_TCPIP_MSG_INPKT);
   if (msg == NULL) {
+    LWIP_ASSERT( "failed to alloc MEMP_TCPIP_MSG_INPKT\n", msg != NULL );
```

Comment:
This is not a place to assert. That brings the system down just because it failed to allocate a buffer for a received packet. It means the system can be crashed by a trivial packet flood attack.

As far as I can see, a failure to allocate here just drops a packet - as long as we eventually get some memory back, and the sender is retransmitting, progress should be made.

Reply:
@kjbracey-arm  yes I agree I will remove the assert.

The UDP packet receive flooding can cause this assert.

Reply:
Yes I will remove this